Project1/project1/import.py

```diff
@@ -0,0 +1,35 @@
+import os
+from flask import Flask, render_template, request
+from flask_sqlalchemy import SQLAlchemy
+import csv
+
+app = Flask(__name__)
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+db = SQLAlchemy(app)
+# db.init_app(app)
+
+from models import *
+
+def main():
+    db.create_all()
```

Comment:
this is redundant as its already done in application.py

Reply:
import.py is a separate python file which i wrote to load the book details into the database as mentioned in LMS. That's why i wronte db.create.all() again in import.py.

Reply:
your already writing db.createall() for users table also, so make sure there it self the books table is also created.

Reply:
Removed db.createall() from import.py and integrated in application.py